### PR TITLE
Feat/liked

### PR DESCRIPTION
**Pull request by ajay-mandaviya**
feat add use like feature 
feat add history page

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -11,8 +12,24 @@ import {
...
 
 function App() {
+  const { dispatch } = useData();
+
+  const {
+    auth: { token },
+  } = useAuth();
+
+  useEffect(() => {
+    console.log("home useEffect");
```
Comment (Manpreet-hub) [inline src/App.js:26]: remove console

File: src/pages/SingleVideo/SingleVideo.jsx [hunk 2]
```diff
@@ -3,38 +3,54 @@ import { useParams } from "react-router-dom";
...
   const [video, setVideo] = useState();
-  // const [categoryVideos, setCategoryVideo] = useState([]);
-  // const getCategoryVideo = (categorys) => {
-  //   const data = videos.filter((video) => video.category === categorys);
-  //   const shufleVideo = shuffleArray(data);
-  //   // setCategoryVideo(shufleVideo);
-  // };
   const getVideo = async () => {
+    console.log("get the videos");
     try {
```
Comment (Manpreet-hub) [inline src/pages/SingleVideo/SingleVideo.jsx:27]: remove console

File: src/services/history.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      type: DataAction.SET_HISTORY_VIDEOS,
+      payload: history,
+    });
+  } catch (error) {
+    console.log("Error while get history", error);
+  }
+};
+export const addHistoryVideo = async (token, video, dispatch) => {
+  console.log("adding video into history");
+  try {
```
Comment (Manpreet-hub) [inline src/services/history.js:23]: +1

**Review by Manpreet-hub — APPROVED** [review]
The code looks good. 
 It would be great if you show some screen recording or working deployed link.

File: src/context/AuthContext.js [hunk 4]
```diff
@@ -25,15 +25,15 @@ const AuthProvider = ({ children }) => {
...
+  }, []);
 
   const loginUser = async (user) => {
     try {
       const {
         data: { encodedToken, foundUser },
         status,
       } = await loginApi(user);
-      //   console.log(data);
+      console.log("slogin status", status);
```
Comment (prathmeshjagtap) [inline src/context/AuthContext.js:36]: remove console.log

File: src/services/history.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+export const getAllHistory = async (token, dispatch) => {
+  try {
+    const {
+      data: { history },
+    } = await axios.get("/api/user/history", {
+      headers: {
+        authorization: token,
+      },
+    });
+    // console.log("histroy videos", data);
```
Comment (prathmeshjagtap) [inline src/services/history.js:12]: remove commented code

File: src/services/likeapi.js [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+    } = await axios.delete(`/api/user/likes/${id}`, {
+      headers: {
+        authorization: token,
+      },
+    });
+    dispatch({
+      type: DataAction.SET_LIKE_VIDEOS,
+      payload: likes,
+    });
+    console.log("data after delte", likes);
```
Comment (prathmeshjagtap) [inline src/services/likeapi.js:60]: +1

**Review by prathmeshjagtap — APPROVED** [review]
The code is clean I liked you have made separate  file for constants' 

Remove commented code and console.log  before merging

**Pull request merged**